_data/rules.yml

```diff
@@ -192,143 +185,146 @@ groups:
                 severity: warning
                 for: 10m
               - name: Host CPU is underutilized
-                description: "CPU load is < 20% for 1 week. Consider reducing the number of CPUs."
-                query: '(100 - (rate(node_cpu_seconds_total{mode="idle"}[30m]) * 100) < 20) * on(instance) group_left (nodename) node_uname_info{nodename=~".+"}'
+                description: "CPU load has been < 20% for 1 week. Consider reducing the number of CPUs."
+                query: '(min by (instance) (rate(node_cpu_seconds_total{mode="idle"}[1h]))) > 0.8'
                 severity: info
-                for: 1w
                 comments: |
                   You may want to increase the alert manager 'repeat_interval' for this type of alert to daily or weekly
+                for: 1w
               - name: Host CPU steal noisy neighbor
                 description: CPU steal is > 10%. A noisy neighbor is killing VM performances or a spot instance may be out of credit.
-                query: '(avg by(instance) (rate(node_cpu_seconds_total{mode="steal"}[5m])) * 100 > 10) * on(instance) group_left (nodename) node_uname_info{nodename=~".+"}'
+                query: 'avg by(instance) (rate(node_cpu_seconds_total{mode="steal"}[5m])) * 100 > 10'
                 severity: warning
               - name: Host CPU high iowait
-                description: CPU iowait > 10%. A high iowait means that you are disk or network bound.
-                query: '(avg by (instance) (rate(node_cpu_seconds_total{mode="iowait"}[5m])) * 100 > 10) * on(instance) group_left (nodename) node_uname_info{nodename=~".+"}'
+                description: CPU iowait > 10%. Your CPU is idling waiting for storage to respond.
+                query: 'avg by (instance) (rate(node_cpu_seconds_total{mode="iowait"}[5m])) > .10'
                 severity: warning
               - name: Host unusual disk IO
-                description: "Time spent in IO is too high on {{ $labels.instance }}. Check storage for issues."
-                query: '(rate(node_disk_io_time_seconds_total[1m]) > 0.5) * on(instance) group_left (nodename) node_uname_info{nodename=~".+"}'
+                description: "Disk usage >80%. Check storage for issues or increase IOPS capabilities."
+                query: 'rate(node_disk_io_time_seconds_total[5m]) > 0.8'
                 severity: warning
-                for: 5m
               - name: Host context switching
                 description: Context switching is growing on the node (> 10000 / CPU / s)
-                query: '((rate(node_context_switches_total[5m])) / (count without(cpu, mode) (node_cpu_seconds_total{mode="idle"})) > 10000) * on(instance) group_left (nodename) node_uname_info{nodename=~".+"}'
+                query: '(rate(node_context_switches_total[5m])) / (count without(cpu, mode) (node_cpu_seconds_total{mode="idle"})) > 10000'
```

Comment:
Why did you remove the left join? The `nodename` lookup is nice for labelling the alert.

(see #348)

Reply:
The uname collector, although enabled by default, may be disabled. Therefore, adding it, although nice for those systems, does result in some environments having a different set of results when it is added. This happens on some  BSD and Docker is weird about it too (if I remember correctly, it returns the information from the container containing the exporter, not the monitored container).

Additionally SystemD out of the box, no longer updates the hostname that uname returns, therefore all of our RHEL boxes were returning localhost.localdomain, despite it having a 'correct' hostname on the network. Since the query wasn't reliable in our environment, we now use a regexp on instance for alert formatting (as in most people's cases, the instance is based on the actual (external) DNS.